grpc-proto/proto/query.proto

```diff
@@ -199,10 +200,25 @@ message ColumnSpec {
   string name = 2;
 }
 
+message Traces {
+  message Event {
+    string activity = 1;
+    string source = 2;
+    int64 source_elapsed = 3;
+    string thread = 4;
+    string event_id = 5;
+  }
+  string id = 1;
+  int64 duration = 2;
+  int64 started_at = 3;
+  repeated Event events = 4;
+}
+
 message Response {
   Payload result_set = 1;
   string tracing_id = 2;
```

Comment:
Should `tracing_id` be removed? It's now in `traces`.

Reply:
That's a good point. I've removed the `tracing_id` from this level. 
However, I need a way to propagate the `tracing_id` from the `handleBatchQuery` and `handleQuery` to  `executeTracingQueryIfNeeded`. For that purpose, I've created a `ResponseAndTraceId` wrapper class.
The con of this approach is that I don't need to transform `UUID -> String -> UUID`, but I can carry over a `UUID` directly.